maps/aurora/aurora-4_mainlevel.dmm

```diff
@@ -6042,8 +6042,8 @@
 	name = "Riot Gear Storage";
 	req_access = list(3)
 	},
-/obj/item/clothing/shoes/leg_guard/riot,
```

Comment:
You are reverting changes made to the map by a prior pr.

Reply:
This code is from a previous PR, please put it back.

Reply:
This was an accident oh god sorry